build.sbt

```diff
@@ -8,7 +8,10 @@ lazy val root = (project in file("."))
     libraryDependencies ++= Seq(
       "org.apache.spark" %% "spark-core" % "3.0.0",
       "org.apache.spark" %% "spark-sql" % "3.0.0",
-      "org.scalatest" %% "scalatest" % "3.2.12" % "test",
-      "com.holdenkarau" %% "spark-testing-base" % "3.4.1_1.5.0" % "test"
+      "org.scalatest" %% "scalatest" % "3.2.9" % "test",
+      "com.holdenkarau" %% "spark-testing-base" % "3.0.0_1.1.0" % "test",
```

Comment:
yeah, in our project we do not use it. We impl. some basic modules in our team, and it's enough to use it.

Reply:
Which ones are the common modules you use day-to-day for testing?